nxviz/plots.py

```diff
@@ -425,32 +435,36 @@ def compute_node_label_positions(self):
         the node coordinates this can be used to add additional annotations
         with rotated text.
         """
-        xs = []
-        ys = []
-        has = []
-        vas = []
-        rotations = []
+        self._init_node_label_meta()
+
         for node in self.nodes:
+
+            # Define radius 'radius' and circumference 'theta'
             theta = node_theta(self.nodes, node)
-            radius = 1.02 * (self.plot_radius + self.nodeprops['radius'])
-            x, y = get_cartesian(r=radius, theta=theta)
-
-            # Computes the text alignment
-            if x == 0:
-                ha = 'center'
-            elif x > 0:
-                ha = 'left'
-            else:
-                ha = 'right'
-            if y == 0:
-                va = 'center'
-            elif y > 0:
-                va = 'bottom'
-            else:
-                va = 'top'
+            # multiplication factor 1.02 moved below
+            radius = self.plot_radius + self.nodeprops['radius']
 
-            if self.rotate:
-                va = "center"
+            # Coordinates of text inside nodes
+            radius_adjustment = 0.96 if self.node_layout == 'numbers' else 1.02
+            x, y = get_cartesian(r=radius * radius_adjustment, theta=theta)
+
+            # For numbered nodes
+
+            # Node description x
+            tx, _ = get_cartesian(r=radius, theta=theta)
+            # Create the quasi-circular positioning on the x axis
+            tx *= 1 - np.log(np.cos(theta) * self.nz_sign(np.cos(theta)))
+            # Move each node a little further away from the circos
+            tx += self.nz_sign(x)
+
+            # Node description y numerator
+            numer = \
+                radius * (theta % (self.nz_sign(y) * self.nz_sign(x) * np.pi))
+            # Node description y denominator
+            denom = (self.nz_sign(x) * np.pi)
+            ty = 2 * (numer / denom)
+
+            # For rotated nodes
```

Comment:
As with `# For numbered nodes`, same point. Perhaps add a few `-` characters before and after to signify that it's like a "heading"?